src/docs/policies.md

```diff
@@ -0,0 +1,59 @@
+# Mission Landing Zone Regulatory Compliance - NIST Policies
```

Comment:
It looks like this document has some ASCII transcription artifacts for single-quote characters throughout: `&#39;`